### ISSUE TITLE
netbeans 8.2 support

### ISSUE THREAD

**Issue by 38leinaD** [tjmcmanus/NetbeansLibertyPlugin#19]
hi,
not sure if it should be working under 8.2. built and installed it myself.
steps:
1. Servers / Add Server
2. Select Websphere Liberty -> Next
3. Installation Location: Select install path of my Websphere Liberty -> Finish
4. Servers / Websphere Liberty / Properties

Actual:
Installation Location is empty. Path was not saved.

Expected:
Installation Location contains Path previously set by me.

Starting the server is also not working. I am thus suspecting that the unsaved path is the reason.

**Comment by GeertjanWielenga** [comment]
Indeed, it would be crazy if it worked, since it's a work in progress right now, right at the beginning phase of development where nothing works yet.

**Comment by 38leinaD** [comment]
well that is good to know :-)
is there active development ongoing/planned in the near future?

**Comment by GeertjanWielenga** [comment]
You can't see that the last updates were during the past months? Confused. And, yes, that means active development.

**Comment by 38leinaD** [comment]
good to know that there is active development. with it, i just meant that there was no commit in the last month.

**Comment by tjmcmanus** [comment]
The last commit was Oct 16th for the Start enabled function
